python/data/taxons.py

```diff
@@ -27,3 +27,11 @@ def content_item_tagged_to_topic_taxonomy(content_item):
             content_item_taxons(content_item)
         )
     )
+
+def content_item_within_part_of_taxonomy(content_item, taxonomy_part_content_id):
```

Comment:
It would be nice to have some tests in this module

Reply:
I've finally got around to having a look at this (see defe95a ), let me know what you think :)